grin/src/adapters.rs

```diff
@@ -88,9 +88,8 @@ impl p2p::ChainAdapter for NetToChainAdapter {
 			b.header.height,
 			addr,
 		);
-
-		self.process_block(b)
-	}
+        self.process_block(b, addr)
```

Comment:
Looks like your indentations are off.

Reply:
Not just here, I'm a bit lost, it looks like 8-spaces tabs are used (I used 4), also do we replace tabs with spaces? Should be fixed in this PR anyway